### PR TITLE
Feature/new card

### PR DESCRIPTION
**Pull request by Veronica2806**

### CODE REVIEW

File: src/components/Article/Article.js [hunk 1]
```diff
@@ -14,11 +13,11 @@ class Article extends Component {
                 title="automatic wi-fi security"
                 />
                 <ArticleText
-                text="mymymymmymymymyymymymmymymymyymymymymyymymymyymymymyymymymymymymm"
+                text="test"
```
Comment (Kronenberg) [inline src/components/Article/Article.js]: test???? remove or?

File: src/components/Article/Article.js [hunk 2]
```diff
@@ -14,11 +13,11 @@ class Article extends Component {
...
                 />
                 <Button title="start now" status={true}/>
                 <Button title="request demo" status={true}/>
```
Comment (Kronenberg) [inline src/components/Article/Article.js]: Create better name for button component, i dont see the differences between regular button.

File: src/components/Card/Card.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+			h1{
+				font: bolder 21px "Arial Black";
+				color: #3a5ba2;
+				line-height: 31px;
+				
+			}
+			
+			img{
+				align-self: center;
+				margin-top: -85px;
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: dont use negative margins

File: src/components/Card/CardItem.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    </li>
+
+);
+
+CardItem.propTypes = {
+    id: PropTypes.number.isRequired,
+    src:PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired
+};
```
Comment (Kronenberg) [inline src/components/Card/CardItem.js:28]: all this props are required are u sure?

File: src/components/Card/Card.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
+.card-list{
+	
+  ul{
+	display: flex;
+    justify-content: space-around;
+    margin: auto;
+    padding:5em;
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: why are u use em?

File: src/components/Card/Card.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+	list-style: none;
+	  
+
+    li{
+		
+		background-color: #fff;
+		border-radius: .3rem;
+		box-shadow: 0 1rem 3rem 0 rgba(96,111,148,.2);
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: put shadow color into scss variable

File: src/components/Card/Card.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    justify-content: space-around;
+    margin: auto;
+    padding:5em;
+	list-style: none;
+	  
+
+    li{
+		
+		background-color: #fff;
+		border-radius: .3rem;
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: rem? why not PX?

File: src/components/Card/Card.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+		box-shadow: 0 1rem 3rem 0 rgba(96,111,148,.2);
+		padding:3em;
+		width: 250px;
+		display: flex;
+		flex-direction: column;
+	
+		
+			p{
+				color: #536283;
+				font: normal 16px "Open Sans";
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: tag font? why not font-family

File: src/components/Article/Article.js (hunk 1 above)
Comment (Veronica2806) [reply src/components/Article/Article.js]: не поняла

File: src/components/Card/Card.scss [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+				line-height: 26px;
+			}
+			h1{
+				font: bolder 21px "Arial Black";
```
Comment (Kronenberg) [inline src/components/Card/Card.scss]: same

File: src/components/Article/Article.js (hunk 1 above)
Comment (Kronenberg) [reply src/components/Article/Article.js]: @Veronika2806 why are u use this property with this value?

Comment (Veronica2806) [reply src/components/Article/Article.js]: ты имеешь ввиду сделать моки?

**Pull request merged**